overlord/assertstate/assertstate_test.go

```diff
@@ -619,6 +619,23 @@ func (s *assertMgrSuite) stateFromDecl(decl *asserts.SnapDeclaration, revno snap
 	})
 }
 
+func (s *assertMgrSuite) stateFromDeclWithName(decl *asserts.SnapDeclaration, instanceName string, revno snap.Revision) {
```

Comment:
Would it make sense to kill the old function above and only keep the new one (in which case it would need to handle empty instanceName argument); alternatively, move the common bits into a helper?